plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/AppEngineJreWhitelistTest.java

```diff
@@ -14,11 +14,13 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.eclipse.appengine.whitelist;
+package com.google.cloud.tools.eclipse.appengine.validation;
 
 import org.junit.Assert;
 import org.junit.Test;
 
+import com.google.cloud.tools.eclipse.appengine.validation.AppEngineJreWhitelist;
```

Comment:
I guess this was added spuriously. I've seen this happening when renaming bundles. Try removing this and reload. This shouldn't be added.